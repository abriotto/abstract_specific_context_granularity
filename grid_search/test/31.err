  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  6%|###########3                                                                                                                                                                                | 6/99 [00:00<00:01, 54.95it/s] 14%|##########################4                                                                                                                                                                | 14/99 [00:00<00:01, 65.38it/s] 22%|#########################################5                                                                                                                                                 | 22/99 [00:00<00:01, 68.64it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:00<00:01, 66.57it/s] 36%|####################################################################                                                                                                                       | 36/99 [00:00<00:00, 65.52it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:00<00:00, 63.45it/s] 51%|##############################################################################################4                                                                                            | 50/99 [00:00<00:00, 63.41it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:00<00:00, 66.43it/s] 69%|################################################################################################################################4                                                          | 68/99 [00:00<00:00, 73.91it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:01<00:00, 67.49it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:01<00:00, 63.81it/s] 91%|##########################################################################################################################################################################                 | 90/99 [00:01<00:00, 61.15it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:01<00:00, 64.37it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 64.54it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:00<00:00, 70.20it/s] 68%|###############################################################################################################################1                                                           | 17/25 [00:00<00:00, 72.36it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 64.22it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 66.10it/s]
