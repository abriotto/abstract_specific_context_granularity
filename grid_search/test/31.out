# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=64", "--learning_rate=0.0005", "--hidden_size=256", "--temp_update=0.97", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.7034088373184204, "acc": 0.5043351650238037, "length": 3.5949041843414307, "mode": "train", "epoch": 1}
{"loss": 0.6946811676025391, "acc": 0.5092592239379883, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6981714963912964, "acc": 0.5089361667633057, "length": 3.3578124046325684, "mode": "train", "epoch": 2}
{"loss": 0.6969155669212341, "acc": 0.49259263277053833, "length": 1.0, "mode": "test", "epoch": 2}
{"loss": 0.6926014423370361, "acc": 0.5257925391197205, "length": 2.6519391536712646, "mode": "train", "epoch": 3}
{"loss": 0.6964030265808105, "acc": 0.5064815282821655, "length": 1.0, "mode": "test", "epoch": 3}
{"loss": 0.688227653503418, "acc": 0.5456632375717163, "length": 2.0564706325531006, "mode": "train", "epoch": 4}
{"loss": 0.6955565214157104, "acc": 0.48611119389533997, "length": 1.0, "mode": "test", "epoch": 4}
{"loss": 0.6796694993972778, "acc": 0.5715946555137634, "length": 2.354205369949341, "mode": "train", "epoch": 5}
{"loss": 0.6786109805107117, "acc": 0.5674846172332764, "length": 1.6166666746139526, "mode": "test", "epoch": 5}
{"loss": 0.6480737328529358, "acc": 0.6449443101882935, "length": 2.5137040615081787, "mode": "train", "epoch": 6}
{"loss": 0.6503287553787231, "acc": 0.6125934720039368, "length": 1.9833333492279053, "mode": "test", "epoch": 6}
{"loss": 0.6048340201377869, "acc": 0.6817651391029358, "length": 2.5641233921051025, "mode": "train", "epoch": 7}
{"loss": 0.6138858199119568, "acc": 0.6709902286529541, "length": 2.5888888835906982, "mode": "test", "epoch": 7}
{"loss": 0.5640014410018921, "acc": 0.7217596173286438, "length": 2.893752336502075, "mode": "train", "epoch": 8}
{"loss": 0.5817922949790955, "acc": 0.7062733769416809, "length": 3.144444465637207, "mode": "test", "epoch": 8}
{"loss": 0.5148265957832336, "acc": 0.7622140049934387, "length": 3.2100648880004883, "mode": "train", "epoch": 9}
{"loss": 0.5545593500137329, "acc": 0.7314971089363098, "length": 3.4000000953674316, "mode": "test", "epoch": 9}
{"loss": 0.4745900332927704, "acc": 0.7930165529251099, "length": 3.4059438705444336, "mode": "train", "epoch": 10}
{"loss": 0.534791111946106, "acc": 0.7761863470077515, "length": 3.3611111640930176, "mode": "test", "epoch": 10}
{"loss": 0.4486672580242157, "acc": 0.8055785298347473, "length": 3.3648993968963623, "mode": "train", "epoch": 11}
{"loss": 0.5310131311416626, "acc": 0.7609999179840088, "length": 3.5888888835906982, "mode": "test", "epoch": 11}
{"loss": 0.43539878726005554, "acc": 0.8039813041687012, "length": 3.424240827560425, "mode": "train", "epoch": 12}
{"loss": 0.5334092378616333, "acc": 0.7686299085617065, "length": 3.366666555404663, "mode": "test", "epoch": 12}
{"loss": 0.4146406352519989, "acc": 0.8189737200737, "length": 3.5276291370391846, "mode": "train", "epoch": 13}
{"loss": 0.49721455574035645, "acc": 0.7834781408309937, "length": 3.6222221851348877, "mode": "test", "epoch": 13}
{"loss": 0.40224796533584595, "acc": 0.8282142281532288, "length": 3.518864393234253, "mode": "train", "epoch": 14}
{"loss": 0.49871760606765747, "acc": 0.793805718421936, "length": 3.594444513320923, "mode": "test", "epoch": 14}
{"loss": 0.3837921619415283, "acc": 0.8297247886657715, "length": 3.584503173828125, "mode": "train", "epoch": 15}
{"loss": 0.46741795539855957, "acc": 0.8006099462509155, "length": 3.7333333492279053, "mode": "test", "epoch": 15}
{"loss": 0.38043588399887085, "acc": 0.8342036008834839, "length": 3.632427930831909, "mode": "train", "epoch": 16}
{"loss": 0.47317036986351013, "acc": 0.795295774936676, "length": 3.6611111164093018, "mode": "test", "epoch": 16}
{"loss": 0.37169328331947327, "acc": 0.8350117206573486, "length": 3.602977752685547, "mode": "train", "epoch": 17}
{"loss": 0.4665636420249939, "acc": 0.8132901787757874, "length": 3.6111111640930176, "mode": "test", "epoch": 17}
{"loss": 0.3602556884288788, "acc": 0.8469841480255127, "length": 3.6282947063446045, "mode": "train", "epoch": 18}
{"loss": 0.46896499395370483, "acc": 0.8226184248924255, "length": 3.711111068725586, "mode": "test", "epoch": 18}
{"loss": 0.35717707872390747, "acc": 0.853360652923584, "length": 3.668168783187866, "mode": "train", "epoch": 19}
{"loss": 0.44549593329429626, "acc": 0.8224002122879028, "length": 3.6111111640930176, "mode": "test", "epoch": 19}
{"loss": 0.34307682514190674, "acc": 0.856377363204956, "length": 3.6786410808563232, "mode": "train", "epoch": 20}
{"loss": 0.4403129518032074, "acc": 0.8335170149803162, "length": 3.722222328186035, "mode": "test", "epoch": 20}
{"loss": 0.3382088243961334, "acc": 0.8622562885284424, "length": 3.723598003387451, "mode": "train", "epoch": 21}
{"loss": 0.44593381881713867, "acc": 0.8323351740837097, "length": 3.6777777671813965, "mode": "test", "epoch": 21}
{"loss": 0.32918885350227356, "acc": 0.8635572791099548, "length": 3.6705801486968994, "mode": "train", "epoch": 22}
{"loss": 0.43783384561538696, "acc": 0.8253171443939209, "length": 3.722222328186035, "mode": "test", "epoch": 22}
{"loss": 0.3196794092655182, "acc": 0.8708776831626892, "length": 3.7535176277160645, "mode": "train", "epoch": 23}
{"loss": 0.42671462893486023, "acc": 0.8358094692230225, "length": 3.7166666984558105, "mode": "test", "epoch": 23}
{"loss": 0.3066892921924591, "acc": 0.872334897518158, "length": 3.7570230960845947, "mode": "train", "epoch": 24}
{"loss": 0.432741641998291, "acc": 0.8332242965698242, "length": 3.700000047683716, "mode": "test", "epoch": 24}
{"loss": 0.3091292381286621, "acc": 0.8741264343261719, "length": 3.7315967082977295, "mode": "train", "epoch": 25}
{"loss": 0.4114559590816498, "acc": 0.8349137306213379, "length": 3.700000047683716, "mode": "test", "epoch": 25}
{"loss": 0.3040980100631714, "acc": 0.8770339488983154, "length": 3.728524684906006, "mode": "train", "epoch": 26}
{"loss": 0.41274839639663696, "acc": 0.8370949625968933, "length": 3.655555486679077, "mode": "test", "epoch": 26}
{"loss": 0.3028625547885895, "acc": 0.8814531564712524, "length": 3.7141082286834717, "mode": "train", "epoch": 27}
{"loss": 0.4172905385494232, "acc": 0.8431468605995178, "length": 3.7055554389953613, "mode": "test", "epoch": 27}
{"loss": 0.2981500029563904, "acc": 0.8770415782928467, "length": 3.7510433197021484, "mode": "train", "epoch": 28}
{"loss": 0.4083118140697479, "acc": 0.843472957611084, "length": 3.7444443702697754, "mode": "test", "epoch": 28}
{"loss": 0.29719874262809753, "acc": 0.8820016384124756, "length": 3.7913002967834473, "mode": "train", "epoch": 29}
{"loss": 0.4405616223812103, "acc": 0.8298355937004089, "length": 3.7333333492279053, "mode": "test", "epoch": 29}
{"loss": 0.29082828760147095, "acc": 0.8821440935134888, "length": 3.7413241863250732, "mode": "train", "epoch": 30}
{"loss": 0.3978360891342163, "acc": 0.8480246663093567, "length": 3.722222328186035, "mode": "test", "epoch": 30}
{"loss": 0.28724002838134766, "acc": 0.8817651271820068, "length": 3.7608742713928223, "mode": "train", "epoch": 31}
{"loss": 0.4237067699432373, "acc": 0.8368790745735168, "length": 3.694444417953491, "mode": "test", "epoch": 31}
{"loss": 0.28693878650665283, "acc": 0.884903073310852, "length": 3.7398314476013184, "mode": "train", "epoch": 32}
{"loss": 0.3967322111129761, "acc": 0.8491164445877075, "length": 3.700000047683716, "mode": "test", "epoch": 32}
{"loss": 0.2762225866317749, "acc": 0.8868023753166199, "length": 3.780574321746826, "mode": "train", "epoch": 33}
{"loss": 0.3891858458518982, "acc": 0.8528245687484741, "length": 3.799999952316284, "mode": "test", "epoch": 33}
{"loss": 0.2638145685195923, "acc": 0.8910542130470276, "length": 3.8164103031158447, "mode": "train", "epoch": 34}
{"loss": 0.381960391998291, "acc": 0.8522592186927795, "length": 3.7166666984558105, "mode": "test", "epoch": 34}
{"loss": 0.2652832269668579, "acc": 0.8922892212867737, "length": 3.7608087062835693, "mode": "train", "epoch": 35}
{"loss": 0.40129002928733826, "acc": 0.8479800820350647, "length": 3.6666667461395264, "mode": "test", "epoch": 35}
{"loss": 0.2609463632106781, "acc": 0.8990271091461182, "length": 3.74729585647583, "mode": "train", "epoch": 36}
{"loss": 0.37300121784210205, "acc": 0.8609151840209961, "length": 3.700000047683716, "mode": "test", "epoch": 36}
{"loss": 0.23874589800834656, "acc": 0.9039095044136047, "length": 3.788876533508301, "mode": "train", "epoch": 37}
{"loss": 0.3584422469139099, "acc": 0.8585671186447144, "length": 3.73888897895813, "mode": "test", "epoch": 37}
{"loss": 0.23493173718452454, "acc": 0.9101108908653259, "length": 3.8054850101470947, "mode": "train", "epoch": 38}
{"loss": 0.3941686749458313, "acc": 0.854932427406311, "length": 3.655555486679077, "mode": "test", "epoch": 38}
{"loss": 0.2407643049955368, "acc": 0.9023762941360474, "length": 3.7295665740966797, "mode": "train", "epoch": 39}
{"loss": 0.37888607382774353, "acc": 0.8522524237632751, "length": 3.700000047683716, "mode": "test", "epoch": 39}
{"loss": 0.2368682473897934, "acc": 0.9041303396224976, "length": 3.7373526096343994, "mode": "train", "epoch": 40}
{"loss": 0.39741408824920654, "acc": 0.8622773885726929, "length": 3.644444465637207, "mode": "test", "epoch": 40}
{"loss": 0.23647113144397736, "acc": 0.9107558727264404, "length": 3.71854567527771, "mode": "train", "epoch": 41}
{"loss": 0.39163321256637573, "acc": 0.8630987405776978, "length": 3.6611111164093018, "mode": "test", "epoch": 41}
{"loss": 0.22969722747802734, "acc": 0.9085155725479126, "length": 3.7405550479888916, "mode": "train", "epoch": 42}
{"loss": 0.36598390340805054, "acc": 0.8655259013175964, "length": 3.7277777194976807, "mode": "test", "epoch": 42}
{"loss": 0.2220381796360016, "acc": 0.9091811776161194, "length": 3.7567522525787354, "mode": "train", "epoch": 43}
{"loss": 0.371040403842926, "acc": 0.8643184304237366, "length": 3.6611111164093018, "mode": "test", "epoch": 43}
{"loss": 0.218020498752594, "acc": 0.920339047908783, "length": 3.736677885055542, "mode": "train", "epoch": 44}
{"loss": 0.3805205523967743, "acc": 0.8662070631980896, "length": 3.683333396911621, "mode": "test", "epoch": 44}
{"loss": 0.2241733819246292, "acc": 0.9093897342681885, "length": 3.785008668899536, "mode": "train", "epoch": 45}
{"loss": 0.3668283224105835, "acc": 0.8644787073135376, "length": 3.76111102104187, "mode": "test", "epoch": 45}
{"loss": 0.22340147197246552, "acc": 0.9140174984931946, "length": 3.7647533416748047, "mode": "train", "epoch": 46}
{"loss": 0.36066776514053345, "acc": 0.8676249980926514, "length": 3.700000047683716, "mode": "test", "epoch": 46}
{"loss": 0.2170470505952835, "acc": 0.9162405729293823, "length": 3.7605395317077637, "mode": "train", "epoch": 47}
{"loss": 0.37026870250701904, "acc": 0.8768708109855652, "length": 3.722222328186035, "mode": "test", "epoch": 47}
{"loss": 0.21718011796474457, "acc": 0.9146226644515991, "length": 3.791810989379883, "mode": "train", "epoch": 48}
{"loss": 0.3672136962413788, "acc": 0.8757623434066772, "length": 3.722222328186035, "mode": "test", "epoch": 48}
{"loss": 0.20194777846336365, "acc": 0.9161156415939331, "length": 3.7704074382781982, "mode": "train", "epoch": 49}
{"loss": 0.3587671220302582, "acc": 0.8782418370246887, "length": 3.7277777194976807, "mode": "test", "epoch": 49}
{"loss": 0.2079152911901474, "acc": 0.9215344786643982, "length": 3.785623073577881, "mode": "train", "epoch": 50}
{"loss": 0.38160741329193115, "acc": 0.8772179484367371, "length": 3.75, "mode": "test", "epoch": 50}
{"loss": 0.2010105401277542, "acc": 0.9201241135597229, "length": 3.794522762298584, "mode": "train", "epoch": 51}
{"loss": 0.36350393295288086, "acc": 0.8722267746925354, "length": 3.711111068725586, "mode": "test", "epoch": 51}
{"loss": 0.20623093843460083, "acc": 0.9248920679092407, "length": 3.7553956508636475, "mode": "train", "epoch": 52}
{"loss": 0.36913108825683594, "acc": 0.8773683309555054, "length": 3.7055554389953613, "mode": "test", "epoch": 52}
{"loss": 0.20335987210273743, "acc": 0.9247377514839172, "length": 3.7579357624053955, "mode": "train", "epoch": 53}
{"loss": 0.36334308981895447, "acc": 0.8821581602096558, "length": 3.6777777671813965, "mode": "test", "epoch": 53}
{"loss": 0.19361945986747742, "acc": 0.9295740723609924, "length": 3.740699529647827, "mode": "train", "epoch": 54}
{"loss": 0.34990426898002625, "acc": 0.8783109784126282, "length": 3.6888887882232666, "mode": "test", "epoch": 54}
{"loss": 0.19757206737995148, "acc": 0.9247446060180664, "length": 3.7615885734558105, "mode": "train", "epoch": 55}
{"loss": 0.3701634407043457, "acc": 0.8758914470672607, "length": 3.6777777671813965, "mode": "test", "epoch": 55}
{"loss": 0.19574280083179474, "acc": 0.9289913773536682, "length": 3.7577168941497803, "mode": "train", "epoch": 56}
{"loss": 0.3551383912563324, "acc": 0.8840478658676147, "length": 3.711111068725586, "mode": "test", "epoch": 56}
{"loss": 0.18881146609783173, "acc": 0.9284135699272156, "length": 3.7718913555145264, "mode": "train", "epoch": 57}
{"loss": 0.3851926326751709, "acc": 0.8847846388816833, "length": 3.7166666984558105, "mode": "test", "epoch": 57}
{"loss": 0.1804794818162918, "acc": 0.9306702017784119, "length": 3.787107229232788, "mode": "train", "epoch": 58}
{"loss": 0.37379324436187744, "acc": 0.879002034664154, "length": 3.777777671813965, "mode": "test", "epoch": 58}
{"loss": 0.1818038523197174, "acc": 0.9264907240867615, "length": 3.8072566986083984, "mode": "train", "epoch": 59}
{"loss": 0.34510812163352966, "acc": 0.884993851184845, "length": 3.73888897895813, "mode": "test", "epoch": 59}
{"loss": 0.1889246255159378, "acc": 0.9287635087966919, "length": 3.796508550643921, "mode": "train", "epoch": 60}
{"loss": 0.358408659696579, "acc": 0.8862324357032776, "length": 3.7166666984558105, "mode": "test", "epoch": 60}
{"loss": 0.18607349693775177, "acc": 0.9315417408943176, "length": 3.7858095169067383, "mode": "train", "epoch": 61}
{"loss": 0.3627227246761322, "acc": 0.8846587538719177, "length": 3.7555556297302246, "mode": "test", "epoch": 61}
{"loss": 0.1849309802055359, "acc": 0.9293047189712524, "length": 3.798893451690674, "mode": "train", "epoch": 62}
{"loss": 0.3705107867717743, "acc": 0.8847812414169312, "length": 3.711111068725586, "mode": "test", "epoch": 62}
{"loss": 0.1783999502658844, "acc": 0.9369687438011169, "length": 3.7615301609039307, "mode": "train", "epoch": 63}
{"loss": 0.36538392305374146, "acc": 0.8819678425788879, "length": 3.672222137451172, "mode": "test", "epoch": 63}
{"loss": 0.1724844127893448, "acc": 0.9319195747375488, "length": 3.762037992477417, "mode": "train", "epoch": 64}
{"loss": 0.3813335597515106, "acc": 0.8804598450660706, "length": 3.683333396911621, "mode": "test", "epoch": 64}
{"loss": 0.1777520328760147, "acc": 0.9345825910568237, "length": 3.780999183654785, "mode": "train", "epoch": 65}
{"loss": 0.36730727553367615, "acc": 0.8820435404777527, "length": 3.7055554389953613, "mode": "test", "epoch": 65}
{"loss": 0.17679652571678162, "acc": 0.9325177073478699, "length": 3.759060859680176, "mode": "train", "epoch": 66}
{"loss": 0.3656882643699646, "acc": 0.8830863833427429, "length": 3.683333396911621, "mode": "test", "epoch": 66}
{"loss": 0.1685837060213089, "acc": 0.9381008148193359, "length": 3.7684364318847656, "mode": "train", "epoch": 67}
{"loss": 0.36170312762260437, "acc": 0.8874366879463196, "length": 3.6777777671813965, "mode": "test", "epoch": 67}
{"loss": 0.1803177148103714, "acc": 0.9323770403862, "length": 3.740211248397827, "mode": "train", "epoch": 68}
{"loss": 0.3567621111869812, "acc": 0.890251100063324, "length": 3.683333396911621, "mode": "test", "epoch": 68}
{"loss": 0.1740437000989914, "acc": 0.9351885318756104, "length": 3.7555222511291504, "mode": "train", "epoch": 69}
{"loss": 0.3711833655834198, "acc": 0.8886474370956421, "length": 3.683333396911621, "mode": "test", "epoch": 69}
{"loss": 0.16946746408939362, "acc": 0.9352651238441467, "length": 3.7560415267944336, "mode": "train", "epoch": 70}
{"loss": 0.38106784224510193, "acc": 0.88023841381073, "length": 3.644444465637207, "mode": "test", "epoch": 70}
{"loss": 0.16644161939620972, "acc": 0.935374915599823, "length": 3.724813938140869, "mode": "train", "epoch": 71}
{"loss": 0.37925633788108826, "acc": 0.8751446008682251, "length": 3.644444465637207, "mode": "test", "epoch": 71}
{"loss": 0.17804700136184692, "acc": 0.9330502152442932, "length": 3.7742440700531006, "mode": "train", "epoch": 72}
{"loss": 0.3924316167831421, "acc": 0.8877983093261719, "length": 3.694444417953491, "mode": "test", "epoch": 72}
{"loss": 0.18072915077209473, "acc": 0.9342230558395386, "length": 3.74225115776062, "mode": "train", "epoch": 73}
{"loss": 0.3705527186393738, "acc": 0.8845654129981995, "length": 3.6777777671813965, "mode": "test", "epoch": 73}
{"loss": 0.1806623488664627, "acc": 0.9325294494628906, "length": 3.770886182785034, "mode": "train", "epoch": 74}
{"loss": 0.408602237701416, "acc": 0.8740740418434143, "length": 3.73888897895813, "mode": "test", "epoch": 74}
{"loss": 0.18254078924655914, "acc": 0.9317164421081543, "length": 3.79371976852417, "mode": "train", "epoch": 75}
{"loss": 0.3942663371562958, "acc": 0.8857082724571228, "length": 3.7055554389953613, "mode": "test", "epoch": 75}
{"loss": 0.16002722084522247, "acc": 0.9367441534996033, "length": 3.7657601833343506, "mode": "train", "epoch": 76}
{"loss": 0.37974292039871216, "acc": 0.8840645551681519, "length": 3.683333396911621, "mode": "test", "epoch": 76}
{"loss": 0.1669849306344986, "acc": 0.9363246560096741, "length": 3.756796360015869, "mode": "train", "epoch": 77}
{"loss": 0.37862488627433777, "acc": 0.8897347450256348, "length": 3.700000047683716, "mode": "test", "epoch": 77}
{"loss": 0.15665431320667267, "acc": 0.9414956569671631, "length": 3.7750635147094727, "mode": "train", "epoch": 78}
{"loss": 0.387706458568573, "acc": 0.8868746161460876, "length": 3.694444417953491, "mode": "test", "epoch": 78}
{"loss": 0.16147300601005554, "acc": 0.938481330871582, "length": 3.7640340328216553, "mode": "train", "epoch": 79}
{"loss": 0.39191222190856934, "acc": 0.8878161311149597, "length": 3.6888887882232666, "mode": "test", "epoch": 79}
{"loss": 0.1589045524597168, "acc": 0.9399749040603638, "length": 3.7704405784606934, "mode": "train", "epoch": 80}
{"loss": 0.4017433524131775, "acc": 0.8790420293807983, "length": 3.722222328186035, "mode": "test", "epoch": 80}
{"loss": 0.17106997966766357, "acc": 0.9383752942085266, "length": 3.783775806427002, "mode": "train", "epoch": 81}
{"loss": 0.413536012172699, "acc": 0.8849071264266968, "length": 3.6777777671813965, "mode": "test", "epoch": 81}
{"loss": 0.15833765268325806, "acc": 0.9457120895385742, "length": 3.740480661392212, "mode": "train", "epoch": 82}
{"loss": 0.3941723704338074, "acc": 0.8851696252822876, "length": 3.6777777671813965, "mode": "test", "epoch": 82}
{"loss": 0.15543706715106964, "acc": 0.9424514770507812, "length": 3.7611937522888184, "mode": "train", "epoch": 83}
{"loss": 0.4050939977169037, "acc": 0.8939015865325928, "length": 3.7333333492279053, "mode": "test", "epoch": 83}
{"loss": 0.14373113214969635, "acc": 0.945070743560791, "length": 3.796173095703125, "mode": "train", "epoch": 84}
{"loss": 0.3901170492172241, "acc": 0.8914174437522888, "length": 3.76111102104187, "mode": "test", "epoch": 84}
{"loss": 0.15469051897525787, "acc": 0.9456202983856201, "length": 3.815023183822632, "mode": "train", "epoch": 85}
{"loss": 0.38607943058013916, "acc": 0.8922677636146545, "length": 3.7722222805023193, "mode": "test", "epoch": 85}
{"loss": 0.1578994244337082, "acc": 0.9427769184112549, "length": 3.7947275638580322, "mode": "train", "epoch": 86}
{"loss": 0.42627406120300293, "acc": 0.8882478475570679, "length": 3.711111068725586, "mode": "test", "epoch": 86}
{"loss": 0.15032602846622467, "acc": 0.9477792382240295, "length": 3.7709579467773438, "mode": "train", "epoch": 87}
{"loss": 0.4148966670036316, "acc": 0.881532609462738, "length": 3.722222328186035, "mode": "test", "epoch": 87}
{"loss": 0.14394521713256836, "acc": 0.9453621506690979, "length": 3.7802486419677734, "mode": "train", "epoch": 88}
{"loss": 0.4037047326564789, "acc": 0.888038694858551, "length": 3.694444417953491, "mode": "test", "epoch": 88}
{"loss": 0.1401822566986084, "acc": 0.9458138346672058, "length": 3.759571075439453, "mode": "train", "epoch": 89}
{"loss": 0.4124375283718109, "acc": 0.8864171504974365, "length": 3.683333396911621, "mode": "test", "epoch": 89}
{"loss": 0.147515669465065, "acc": 0.9469818472862244, "length": 3.7601702213287354, "mode": "train", "epoch": 90}
{"loss": 0.45055240392684937, "acc": 0.8792734742164612, "length": 3.6777777671813965, "mode": "test", "epoch": 90}
{"loss": 0.15770138800144196, "acc": 0.9452400803565979, "length": 3.749070644378662, "mode": "train", "epoch": 91}
{"loss": 0.4445401430130005, "acc": 0.8830317854881287, "length": 3.7333333492279053, "mode": "test", "epoch": 91}
{"loss": 0.15981239080429077, "acc": 0.9407793283462524, "length": 3.78857159614563, "mode": "train", "epoch": 92}
{"loss": 0.4786011576652527, "acc": 0.8793904781341553, "length": 3.7444443702697754, "mode": "test", "epoch": 92}
{"loss": 0.1675107479095459, "acc": 0.9411214590072632, "length": 3.7903318405151367, "mode": "train", "epoch": 93}
{"loss": 0.42883366346359253, "acc": 0.8776242733001709, "length": 3.73888897895813, "mode": "test", "epoch": 93}
{"loss": 0.16562098264694214, "acc": 0.9349309206008911, "length": 3.7798120975494385, "mode": "train", "epoch": 94}
{"loss": 0.4379940927028656, "acc": 0.8730513453483582, "length": 3.6888887882232666, "mode": "test", "epoch": 94}
{"loss": 0.16330814361572266, "acc": 0.9392380118370056, "length": 3.756643056869507, "mode": "train", "epoch": 95}
{"loss": 0.4119546413421631, "acc": 0.8816896080970764, "length": 3.683333396911621, "mode": "test", "epoch": 95}
{"loss": 0.15629100799560547, "acc": 0.9406132102012634, "length": 3.7585372924804688, "mode": "train", "epoch": 96}
{"loss": 0.41816169023513794, "acc": 0.8833979368209839, "length": 3.672222137451172, "mode": "test", "epoch": 96}
{"loss": 0.15061922371387482, "acc": 0.9417384266853333, "length": 3.752121686935425, "mode": "train", "epoch": 97}
{"loss": 0.4123642146587372, "acc": 0.883134126663208, "length": 3.6888887882232666, "mode": "test", "epoch": 97}
{"loss": 0.14556550979614258, "acc": 0.9467577338218689, "length": 3.773442506790161, "mode": "train", "epoch": 98}
{"loss": 0.40151485800743103, "acc": 0.8842203617095947, "length": 3.75, "mode": "test", "epoch": 98}
{"loss": 0.14633628726005554, "acc": 0.9488896727561951, "length": 3.7891204357147217, "mode": "train", "epoch": 99}
{"loss": 0.3959213197231293, "acc": 0.8937221765518188, "length": 3.73888897895813, "mode": "test", "epoch": 99}
{"loss": 0.13395681977272034, "acc": 0.9507769346237183, "length": 3.7976419925689697, "mode": "train", "epoch": 100}
{"loss": 0.4178975820541382, "acc": 0.8893173336982727, "length": 3.7722222805023193, "mode": "test", "epoch": 100}
test accuracy: 0.7649360299110413
